libeos-updater-util/avahi-service-file.h

```diff
@@ -44,4 +46,42 @@ gboolean eos_avahi_service_file_delete (const gchar   *avahi_service_directory,
                                         GCancellable  *cancellable,
                                         GError       **error);
 
+/* new DNS-SD records format for ostree */
```

Comment:
Eventually it would be good to have some documentation for the format somewhere (in a comment, probably), to save people having to read through the code and work out all the invariants.